.circleci/get_data.sh

```diff
@@ -67,10 +67,18 @@ run_xcpd_cmd () {
     fi
 
     # Otherwise we're going to use docker from the outside
-    bids_mount="-v ${bids_dir}:/bids-input:ro"
+    bids_parent_dir="$(dirname "$bids_dir")"  # get parent directory
+    if [[ ${DS} = nibabies ]]; then
+      bids_folder_name="nibabies"  # get folder name
+    else
+      bids_folder_name="fmriprep"
+    fi
```

Comment:
Does "basename" not work? I'm concerned that hardcoding the paths here could be a problem when we expand to testing DCAN and HCP inputs.

Reply:
basename wasn't working

Reply:
Do you know what the specific problem is? Is it something in how I was calling basename (I haven't used it before), or is it a problem with basename itself?

Reply:
I think basename was just defaulting to 'fmriprep'

Reply:
Something must have been wrong with the path I called it on or something, because I don't think basename can produce "fmriprep" out of nothing.

Reply:
Yes - I'm not sure how to update it. Would you have any recommendations?

Reply:
Can you echo the path and run basename on it locally to see what comes out?

Reply:
It works now